install/src/command.rs

```diff
@@ -1283,3 +1283,35 @@ pub fn run(
         }
     }
 }
+
+pub fn list(config_file: &str) -> Result<(), String> {
+    let config = Config::load(config_file)?;
+
+    let entries = fs::read_dir(&config.releases_dir).map_err(|err| format!("Failed to read install directory, double check that your configuration file is correct: {err}"))?;
+
+    for entry in entries {
+        match entry {
+            Ok(entry) => {
+                let entry_path = entry.path();
+                let dir_name = entry_path
+                    .file_name()
+                    .map(|name| name.to_string_lossy())
+                    .unwrap_or_default();
```

Comment:
```suggestion
                let dir_name = entry
                    .file_name()
                    .to_string_lossy();
```
Can skip some extra conversion and unwrap